playbooks/sudorule/ensure-sudorule-is-absent.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Tests
+  hosts: ipaserver
+  become: true
+
+  tasks:
+  # Ensure sudorule command is present
+  - ipasudorule:
+      ipaadmin_password: pass1234
+      name: testrule1
+      description: A test sudo rule.
```

Comment:
description is not allowed for state `absent`.

Reply:
fixed.